### PR TITLE
README.md: Add badges and fix links

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by attachmentgenie — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # puppet-nomad
 
+[![Build Status](https://travis-ci.com/voxpupuli/puppet-nomad.svg?branch=master)](https://travis-ci.com/voxpupuli/puppet-nomad)
```
Comment (alexjfisher) [inline README.md:3]: We don't use travis anymore...

**Review by alexjfisher — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (attachmentgenie) [reply README.md:3]: good catch, i missed that one

File: README.md [hunk 2]
```diff
@@ -17,9 +25,9 @@ See [REFERENCE](REFERENCE.md).
 Depends on the JSON gem, or a modern ruby. (Ruby 1.8.7 is not officially supported)
```
Comment (ekohl) [inline README.md:25]: For a future PR: we shouldn't mention Ruby 1.8.7 ;)

File: README.md (hunk 1 above)
Comment (bastelfreak) [reply README.md]: fixed it

**Review by alexjfisher — APPROVED** [review]

**Pull request merged**